shared/tracker/action.render.js.flow

```diff
@@ -6,11 +6,13 @@ export type ActionProps = {
   loggedIn: boolean,
   state: SimpleProofState,
   currentlyFollowing: boolean,
-  username: ?string,
+  username: string,
```

Comment:
Not sure why username allowed null, seems like we should always know it.  Anyone know?